src/scenes/AcceleratorRouter/Cohorts/index.tsx

```diff
@@ -10,10 +10,10 @@ import CohortsListView from './CohortsListView';
 const CohortsRouter = () => {
   return (
     <Routes>
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS_EDIT} element={<CohortEditView />} />
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS_NEW} element={<CohortNewView />} />
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS_VIEW} element={<CohortView />} />
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS} element={<CohortsListView />} />
+      <Route path={':cohortId/edit'} element={<CohortEditView />} />
+      <Route path={'new'} element={<CohortNewView />} />
+      <Route path={':cohortId'} element={<CohortView />} />
+      <Route path={''} element={<CohortsListView />} />
```

Comment:
You can put * here and remove next line

Reply:
No we need the navigate-to UX so url is updated as well.

Reply:
Oh, true